### ISSUE TITLE
Should `=> { return` be bad text?

### ISSUE THREAD

**Issue by zepumph** [phetsims/chipper#790]
IMO one lined arrow functions that return something should not use the curly braces and `return` keyword, because it is simpler to just have `arg => arg === 3` instead of  `arg => { return arg === 3; }`. 

If it is a more complex arrow function, then perhaps it can be on more than one line, and then include those pieces. There are currently 33 usages of ` => { return `.

Marking for developer meeting.

**Comment by samreid** [comment]
All of the existing 33 cases should be converted, they will be more readable without the `{return ;}`.  I also think the lint rule would be a good idea, thanks!

**Comment by jonathanolson** [comment]
I could imagine having a filter arrow function where it's one expression to return BUT really needs to take up multiple lines. I find the braces to be helpful for multi-line arrow functions, but I would prefer the one-liners (brace, space, `return`) to be converted.

**Comment by pixelzoom** [comment]
Should `arg =>` also be preferred over `( arg ) =>` ?

**Comment by samreid** [comment]
Here's an eslint rule that may help us:
https://eslint.org/docs/rules/arrow-parens

**Comment by pixelzoom** [comment]
@jonathanolson said:
> I could imagine having a filter arrow function where it's one expression 

Something like this:

```js
arg => {
  return expression1 && expression2 && expression3 && .... && expressionN;
} 
```

can be written without curly braces as:

```js
arg => ( expression1 && expression2 && expression3 && .... && expressionN )
```

**Comment by zepumph** [comment]
From this issue will investigate two lint rules:

- [ ] `arrow-parens` with the "as-needed" option looked just like what we want, where it will error our for single args that include parens.
- [ ] https://eslint.org/docs/rules/arrow-body-style will likely help us with one liner returns.

**Comment by zepumph** [comment]
There were ~160 issues with `arrow-parens "as-needed"`.  The fixes are above, lint rule commit coming in shortly.

I looked into using `arrow-body-style` "as-needed" but it trying to convert too many arrow functions into complicated one-liners that were 160+ chars long (about 150 of them). So instead I just manually added the namesake of this issue as bad text. There were 30 usages of that. 

The rules are now committed and lint is passing. Closing
